ui/src/shared/actions/me.ts

```diff
@@ -25,8 +27,12 @@ export const getMe = () => async dispatch => {
   try {
     const user = await client.users.me()
 
+    HoneyBadger.setContext({
+      user_id: user.id,
+    })
+
     dispatch(setMe(user))
-  } catch (e) {
-    console.error(e)
+  } catch (error) {
+    console.error(error)
```

Comment:
It looks like you're importing `reportError` but not using it in the `catch`, did we want to report the error for this?

https://github.com/influxdata/influxdb/pull/16278/files#diff-023642cbf29399507646a13659e14d24R3

Reply:
@Asalem1 i had `reportError` being called there, but took it out because that introduces a separate logical change to this PR beyond adding honeybadger context and that's something i'm trying to do less of.